src/Helpers/api.js

```diff
@@ -1,6 +1,6 @@
 import axios from "axios";
 
 export default axios.create({
-    // baseURL: "http://localhost:8000/api/",
-    baseURL: "https://dry-tor-16883.herokuapp.com/api/",
+    baseURL: "http://localhost:8000/api/",
```

Comment:
when you are working locally you can change this file to localhost and then just make sure you don't add it to a commit since you want the liveVersion to continue to use the heroku api